src/TypeScript/typescript.js

```diff
@@ -0,0 +1,7 @@
+var myName = function (person) {
+  return "Hello, " + person;
+};
+//myName function that expects an arugument as a String
+var user = "Harman Kalair";
+console.log(myName(user));
+//# sourceMappingURL=typescript.js.map
```

Comment:
Since this is the output of a build, it's what's known as a build artifact (it can be automatically generated by running `npm run build` - it wasn't written by you). Same for the `.js.map` file you have below. It's standard to exclude these from source control so try adding `.js` and `.js.map` to the gitignore for only this directory and removing them from source control.